AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/copy-join-link/copy-join-link.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit, ViewChild, ElementRef, Input } from '@angular/core';
+import { ClipboardService } from 'ngx-clipboard';
+import { CopyDetailsBase } from '../component/copy-details-base';
+import { PageUrls } from 'src/app/shared/page-url.constants';
+import { ConfigService } from '../../services/config.service';
+
+@Component({
+    selector: 'app-copy-join-link',
+    templateUrl: './copy-join-link.component.html',
+    styleUrls: ['./copy-join-link.component.scss']
+})
+export class CopyJoinLinkComponent extends CopyDetailsBase implements OnInit {
+    @Input() set quickLinkDetails(value: string) {
+        this._detailsToCopy = this.vh_video_uri + [PageUrls.JoinLink] + '/' + value;
+    }
+    @ViewChild('conferenceJoinByLink', { static: false }) conferenceJoinByLink: ElementRef;
+
+    private readonly vh_video_uri: string;
+
+    constructor(protected clipboardService: ClipboardService, private configService: ConfigService) {
+        super(clipboardService);
+        this.vh_video_uri = this.configService.getConfig().video_web_url;
+    }
+
+    ngOnInit(): void {
+        this.displayTooltip = true;
```

Comment:
Rename displayTooltip to hideTooltip as displayTooltip = true actually hides the tooltip so it can be confusing.

Reply:
thats done now on last commit